### PR TITLE
Add `remove_feed_item` example

### PR DESCRIPTION
**Pull request by BenRKarl**

### CODE REVIEW

File: examples/feeds/remove_feed_items.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+        )
+        operations.append(feed_item_operation)
+
+    try:
+        response = feed_item_service.mutate_feed_items(customer_id, operations)
+
+        for feed_item in response.results:
+            print(
+                f"Removed feed item with resource name: "
+                "'{feed_item.resource_name}'"
```
Comment (AndrewMBurke) [inline examples/feeds/remove_feed_items.py:56]: F-string out of place, should be:
```
"Removed feed item with resource name: "
f"'{feed_item.resource_name}'"
```

**Review by AndrewMBurke — APPROVED** [review]
LGTM with one minor fix.

File: examples/feeds/remove_feed_items.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    feed_item_service = client.get_service("FeedItemService", version="v5")
+
+    operations = []
+    for feed_item_id in feed_item_ids:
+        # Constructs an operation that will remove the feed item based on the
+        # resource name.
+        feed_item_operation = client.get_type("FeedItemOperation", version="v5")
+        feed_item_operation.remove = feed_item_service.feed_item_path(
+            customer_id,
+            # constructs a resource name for a feed_item, which is in the
```
Comment (wihl) [inline examples/feeds/remove_feed_items.py:44]: nit: s/constructs/Constructs

File: examples/feeds/remove_feed_items.py [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+        "-c",
+        "--customer_id",
+        type=str,
+        required=True,
+        help="The Google Ads customer ID.",
+    )
+    parser.add_argument(
+        "-f",
+        "--feed_id",
+        type=str,
```
Comment (wihl) [inline examples/feeds/remove_feed_items.py:90]: I wonder if we should make the feed_id and the feed_items_ids as integers insteads of strs so as to enforce better typing at the arg level

File: examples/feeds/remove_feed_items.py (hunk 2 above)
Comment (wihl) [inline examples/feeds/remove_feed_items.py:44]: This comment needs to be moved up two lines as `.feed_item_path` creates the full resource name whereas `.format_composite` only creates `{feed_id}~{feed_item_id}`

**Review by wihl — APPROVED** [review]
LGTM with minor comments and suggestions

File: examples/feeds/remove_feed_items.py (hunk 1 above)
Comment (BenRKarl) [reply examples/feeds/remove_feed_items.py:56]: Done

File: examples/feeds/remove_feed_items.py (hunk 3 above)
Comment (BenRKarl) [reply examples/feeds/remove_feed_items.py:90]: Let's discuss separately in the context of the other examples and other libraries.

File: examples/feeds/remove_feed_items.py (hunk 2 above)
Comment (BenRKarl) [reply examples/feeds/remove_feed_items.py:44]: Done

**Pull request merged**